### PR TITLE
1.1 Move files from MediaBundle to MediaAdminBundle

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/94
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/139

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: MediaAdmin/EventSubscriber/DeleteMediaSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return array(
+            MediaEvents::MEDIA_DELETE => 'deleteMedia',
+            KernelEvents::TERMINATE => 'removeMedias',
+        );
+    }
+
```
Comment (alavieille) [inline MediaAdmin/EventSubscriber/DeleteMediaSubscriber.php]: remove this empty line

File: MediaAdmin/Thumbnail/Strategies/ImageToThumbnailManager.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+{
+    const MIME_TYPE_FRAGMENT_IMAGE = 'image';
+
+    protected $tmpDir;
+    protected $dispatcher;
+
+    /**
+     * @param $tmpDir
+     */
+    public function __construct($tmpDir, $dispatcher)
```
Comment (alavieille) [inline MediaAdmin/Thumbnail/Strategies/ImageToThumbnailManager.php]: update phpdoc

File: MediaAdminBundle/Manager/SaveMediaManager.php [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+     * @param DocumentManager           $documentManager
+     * @param FolderRepositoryInterface $folderRepository
+     * @param string                    $mediaClass
+     */
+    public function __construct(
+        $tmpDir,
+        ThumbnailManager $thumbnailManager,
+        UploadedMediaManager $uploadedMediaManager,
+        array $allowedMimeTypes,
+        DocumentManager $documentManager,
```
Comment (alavieille) [inline MediaAdminBundle/Manager/SaveMediaManager.php]: not use DocumentManager but ObjectManager

File: MediaAdminBundle/Manager/SaveMediaManagerInterface.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+     * @param UploadedFile $uploadedFile
+     * 
+     * @return string|null
+     */
+    public function getFilenameFromChunks(UploadedFile $uploadedFile);
+
+    /**
+     * Return true if the file is allowed to be uploaded based on its mime type
+     * 
+     * @param $filename
```
Comment (alavieille) [inline MediaAdminBundle/Manager/SaveMediaManagerInterface.php]: add type in param

File: MediaAdminBundle/Tests/Manager/ImageResizerManagerTest.php [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    public function testCrop($x, $y, $h, $w, $format)
+    {
+        if (file_exists($this->tmpDir .'/' . $format . '-' . $this->file)) {
+            unlink($this->tmpDir .'/' . $format . '-' . $this->file);
+        }
+        $this->assertFileNotExists($this->tmpDir .'/' . $format . '-' . $this->file);
+
+        $this->manager->crop($this->media, $x, $y, $h, $w, $format);
+
+//        TODO Check why travis sees two different strings
```
Comment (alavieille) [inline MediaAdminBundle/Tests/Manager/ImageResizerManagerTest.php]: remove this comment lines

**Comment by ngilain** [comment]
@itkg-alavieille : update done

File: MediaAdmin/Thumbnail/Strategies/ImageToThumbnailManager.php [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+ */
+class ImageToThumbnailManager implements ThumbnailInterface
+{
+    const MIME_TYPE_FRAGMENT_IMAGE = 'image';
+
+    protected $tmpDir;
+    protected $dispatcher;
+
+    /**
+     * @param $tmpDir
```
Comment (alavieille) [inline MediaAdmin/Thumbnail/Strategies/ImageToThumbnailManager.php]: update phpdoc

**Comment by alavieille** [comment]
else phpdoc :+1:

**Pull request merged**